devel/validate-platforms

```diff
@@ -40,20 +40,41 @@ main() {
         echo "[$platform] Generating report file: $report"
         mkdir -p "$(dirname "$report")"
 
+        # Versions to track from `pip list`
+        tracked_packages=(bcbio-gff
+            biopython
+            cycler
+            evofr
+            nextstrain-augur
+            nextstrain-cli
+            numpy
+            numpyro
+            pandas
+            phylo-treetime
+            pyfastx
+            pysam
+            rethinkdb
+            scikit-learn
+            scipy
+            snakemake
+            xopen
+        )
```

Comment:
There's some duplication in this script now (it checks versions of Augur and CLI twice). If adding this change, I would factor the code a bit more to track non-pip and pip-installed packages separately.

Reply:
I'm not sure if `pip list` just shows `pip` installed packages. I think it shows all Python packages. Even if `mamba` installed. But we may not mamba install so we may well have pip-installed them.

Agree that it could be refactored. Just wanted to throw the very useful pip list output in the ring. I thought I'd written an issue to ask for `pip list` output somewhere, maybe `conda-base`, a while ago.

Reply:
> I don't think so 🙃 have a look at the full diff

Oops, I don't know how I skipped over that. Disregard!